src/menu/views/FilterMenu.js

```diff
@@ -25,6 +25,22 @@ const FilterMenu = MenuBuilder.extend({
       return this.g.columns.set("hidden", hidden);
     });
 
+    this.addNode("Hide non-polymorphic sites",(e) => {
+      var threshold = prompt("Enter identity threshold (in percent)", 100);
+      threshold = threshold / 100;
+      var maxLen = this.model.getMaxLength();
+      var hidden = [];
+      // TODO: cache this value
+      var conserv = this.g.stats.scale(this.g.stats.conservation());
+      var end = maxLen - 1;
+      for (var i = 0; 0 < end ? i <= end : i >= end; 0 < end ? i++ : i--) {
```

Comment:
those weird loops are an artifact from the translation of the project from CoffeeScript to ES6.

In modern JS you can write this:

```
conserv.forEach((el, i) => {
    if (el >= threshold)
      hidden.push(i);
})
```

the arrow function will be transpiled with babel, forEach is ES5 standard ;-)